### DISCUSSION TITLE
How to make workflow files up to date?

### DISCUSSION THREAD

**Discussion by jeyraof** [upptime/upptime#96] [Q&A] [answered]
/question

My project's action is [failing](https://github.com/jeyraof/otzil-upptime/actions/runs/390575852) every morning.

So I want to try update workflow files for fixing it.

Is there any way to suggest?

**Comment by AnandChowdhary** [comment, marked as answer]
In your "Uptime Template CI", workflow, please change:

```
actions/setup-node@v2.1.1
```

to

```
actions/setup-node@v1.4.4
```

and then manually run "Update Template CI".

**Comment by jeyraof** [comment]
Solved! Thank you @AnandChowdhary
